client/client.go

```diff
@@ -78,6 +81,9 @@ type Client struct {
 	basePath string
 	// client used to send and receive http requests.
 	client *http.Client
+	// dialer used for hijacking. When set to non-nil, it typically corresponds to
+	// client.Transport.(*http.Tranport).DialContext .
```

Comment:
"When set to non-nil, it typically corresponds to client.Transport.(*http.Tranport).DialContext" seems superfluous.  Better to document appropriate values on the exported method that allows someone to set the value.